app/models/episode.rb

```diff
@@ -67,7 +67,8 @@ class Episode < ApplicationRecord
   scope :draft, -> { where("episodes.published_at IS NULL") }
   scope :scheduled, -> { where("episodes.published_at IS NOT NULL AND episodes.published_at > now()") }
   scope :draft_or_scheduled, -> { draft.or(scheduled) }
-  scope :after, ->(time) { where("#{DROP_DATE} > ?", time) }
+  scope :after, ->(time) { where("#{DROP_DATE} >= ?", time) }
```

Comment:
This was only used in one place in the UI - so  shouldn't matter that I changed this from exclusive to inclusive.